test/server.test.js

```diff
@@ -0,0 +1,27 @@
+const request = require('supertest');
+const app = require('../app')
+describe('Test the Server Connection', () => {
+    test('It should respond to the GET method', (done) => {
+        request(app).get('/').then((response) => {
```

Comment:
As spoken, I think it may be helpful for you to also do some unit tests in addition to these integration tests.